app/src/components/learningJourney/LearningJourneyTile.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import "flowbite";
+
+function LearningJourneyTile({ learningId, jobName, jobDesc, isJobActive }) {
+  if (!isJobActive) {
+    return (null);
+  }
+
+  return (
+    <div className="max-w-sm bg-white rounded-lg border border-gray-100 hover:shadow-lg hover:shadow-orange-200 mt-10 mx-5">
```

Comment:
Possible to refactor this into smaller components but in the same file? Something like jobTile